zanata-war/src/main/java/org/zanata/action/MergeTranslationsManager.java

```diff
@@ -0,0 +1,113 @@
+package org.zanata.action;
+
+import java.io.Serializable;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import org.jboss.seam.ScopeType;
+import org.jboss.seam.annotations.AutoCreate;
+import org.jboss.seam.annotations.In;
+import org.jboss.seam.annotations.Name;
+import org.jboss.seam.annotations.Scope;
+import org.jboss.seam.security.Identity;
+import org.zanata.async.AsyncTaskHandleManager;
+import org.zanata.async.handle.MergeTranslationsTaskHandle;
+import org.zanata.service.MergeTranslationsService;
+
+/**
+ * Manages copy translations from existing version to another tasks.
+ *
+ * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
+ */
+@AutoCreate
+@Name("mergeTranslationsManager")
+@Scope(ScopeType.STATELESS)
+@Slf4j
+public class MergeTranslationsManager implements Serializable {
+    @In
+    private AsyncTaskHandleManager asyncTaskHandleManager;
+
+    @In
+    private MergeTranslationsService mergeTranslationsServiceImpl;
+
+    @In
+    private Identity identity;
+
+    /**
+     * Merge translations from an existing version to another.
+     *
+     * @param sourceProjectSlug - source project identifier
+     * @param sourceVersionSlug - source version identifier
+     * @param targetProjectSlug - target project identifier
+     * @param targetVersionSlug - target version identifier
```

Comment:
This is like saying having a comment of `check if string is empty` in `StringUtils.isStringEmpty()` method if waste of space.